src/pages/Profile.tsx

```diff
@@ -21,45 +23,176 @@ import { SubscriptionCollection } from "../../generated/definitions/backend/Subs
 import { SubscriptionContract } from "../../generated/definitions/backend/SubscriptionContract";
 import { UserData } from "../../generated/definitions/backend/UserData";
 
-import { ValidService } from "../utils/service";
+import {
+  getServiceReviewStatus,
+  ServiceStatus,
+  ValidService
+} from "../utils/service";
 
 const getMail = (email: string) =>
   email && email !== "" ? atob(email) : undefined;
 
 const SubscriptionService = ({
   service,
-  t
+  t,
+  status,
+  atLeastOneVisible
 }: {
   service: Service;
   t: (key: string) => string;
+  status: string;
+  atLeastOneVisible: boolean;
 }) => {
   return service ? (
     <div>
-      <h5>{service.service_id}</h5>
-      <div>
-        {t("service:name")}: {service.service_name}
-      </div>
-      <div>
-        {t("service:department")}: {service.department_name}
+      <h5>
+        <span className="light-text">{t("service:title")}:</span>{" "}
+        <span className="dark-text">{service.service_id}</span>
+      </h5>
+      <div className="my-3">
+        <span className="light-text">{t("service:name")}:</span>{" "}
+        <span className="light-text">{service.service_name}</span>
       </div>
-      <div>
-        {t("service:organization")}: {service.organization_name}
+      <div className="my-3">
+        <span className="light-text">{t("service:organization")}:</span>{" "}
+        <span className="light-text">{service.organization_name}</span>
       </div>
-      <div>
-        {t("service:organization_fiscal_code")}:{" "}
-        {service.organization_fiscal_code}
```

Comment:
Is `organization_fiscal_code` removed?

Reply:
Yes, it was removed from new UI redesign by Roberta.